### PR TITLE
Bill luu/send lights

### PR DESCRIPTION
**Pull request by bill-luu**

### CODE REVIEW

File: EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c [hunk 1]
```diff
@@ -19,6 +20,7 @@
 #define BATTERY_FAULTS_LENGTH (3)
 #define BATTERY_DETAILS_LENGTH (51)
 #define MPPT_DETAILS_LENGTH (13)
+#define LIGHTS_DETAILS_LENGTH (7)
```
Comment (JordanHeinrichs) [inline EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c]: The message has only 3 bytes of content. While this matches the protocol I think the protocol is wrong here.

Comment (bill-luu) [inline EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c]: I think having the 3 uchars makes sense, unless we're missing anything in the protocol @enochtsang ?

Comment (enochtsang) [inline EpsilonCCS/Src/TelemetryReporting/TelemetryReporting.c]: Yea, 3 bytes content but +4 overhead

**Comment by JordanHeinrichs** [comment]
The merge conflicts resolution looks good, the lights packet length still should be looked at. It doesn't seem right for it to be 7 bytes long when there is only 3 bytes of content.

**Comment by bill-luu** [comment]
Yeah you're right, will fix tomorrow.

**Pull request merged**